client/landing/gutenboarding/onboarding-block/edit.tsx

```diff
@@ -49,20 +48,24 @@ const OnboardingEdit: FunctionComponent< BlockEditProps< Attributes > > = ( {
 						/>
 						{ siteVertical && (
 							<div className="onboarding-block__footer">
-								<Button className="onboarding-block__question-skip" isLink>
+								<Link
+									to={ Step.DesignSelection }
+									className="onboarding-block__question-skip"
+									isLink
+								>
 									{ /* @TODO: add transitions and correct action */ }
-									{ siteTitle ? NO__( 'Continue' ) : NO__( "Don't know yet" ) } →{ ' ' }
-								</Button>
+									{ siteTitle ? NO__( 'Continue' ) : NO__( "Don't know yet" ) } →
+								</Link>
 							</div>
 						) }
 					</div>
 				</div>
-			);
-		case Steps.DesignSelection:
-			return <DesignSelector />;
-	}
-
-	return null;
+			</Route>
+			<Route path={ Step.DesignSelection }>
```

Comment:
`exact` like the other?
```suggestion
			<Route path={ Step.DesignSelection } exact>
```

Reply:
:man_shrugging: Not sure. It means matching `/design` and only `/design` but not `/design/blah` -- but since we don't have that, we don't really care. (For the root `/`, it's obviously relevant.)

Inclined to go without `exact`.

Reply:
Sounds good 👍

Reply:
Sounds good 👍